src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,38 @@
 import React from 'react';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+import { Comments } from '../../types/Comments';
+
+type Props = {
+  comment: Comments;
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => {
+  const {
+    name,
+    email,
+    body,
+  } = comment;
+
+  return (
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">
+          {name}
+        </strong>
+
+        {' by '}
+
```

Comment:
wil be better (Ihor fixed it today on the lesson): 
        `<span>`
         ` by`
        `</span>`

Reply:
✔️